app/src/main/java/mollie/tictactoe/activities/BoardActivity.java

```diff
@@ -0,0 +1,84 @@
+package mollie.tictactoe.activities;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+
+import mollie.tictactoe.GameHelper;
+import mollie.tictactoe.R;
+import mollie.tictactoe.StateManager;
+import mollie.tictactoe.ui.BoardView;
+import mollie.tictactoe.ui.CellButton;
+import mollie.tictactoe.ui.UIBoardManager;
+
+public class BoardActivity extends AppCompatActivity {
```

Comment:
I'm not quite sure I understand what `role` this class plays in the overall picture.
Is `BoardActivity` the most fitting name? If we had no tech at all -only pens and paper- how would you explain this class or its _role_ to your non-tech relative?